pkg/config/template/renderer_test.go

```diff
@@ -107,6 +107,24 @@ func TestRender(t *testing.T) {
 			`{ "key": the-key }`,
 			false,
 		},
+		{
+			"renders simple template containing three subsequent {",
+			&InMemoryTemplate{
+				content: "{{{ .val }}}",
+			},
+			map[string]interface{}{"val": "the-key"},
+			`{the-key}`,
+			false,
+		},
+		{
+			"renders simple template containing multiple three subsequent {",
+			&InMemoryTemplate{
+				content: "{{{ .val1 }}} {{{ .val2 }}}",
+			},
+			map[string]interface{}{"val1": "the-key1", "val2": "the-key2"},
+			`{the-key1} {the-key2}`,
+			false,
+		},
```

Comment:
could you please add a few more test cases? 

1. The full sample that found this issue `{{{.val}} | bearer}` -> `{the-key | bearer}`
2. Some non-template cases that are not broken by this change like `\\{\\{\\{ I just like curlies` -> `{{{ I just like curlies`